lib/index.js

```diff
@@ -12,6 +12,11 @@ const Celebrate = (schema, options) => {
   Assert.ifError(result.error);
   const rules = new Map();
 
+  if (options === undefined) {
```

Comment:
Couldn't this just be:

`options = Object.assign({ escapeHtml: true }, options)`

Reply:
Indeed, I don't know why I was pretty sure you could not assign something to undefined, but in fact we can